src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -68,49 +74,53 @@ const MediaPlayer = ({ enableFileDownload = false, enablePIP = false }) => {
   }, [manifest, canvasIndex, srcIndex]); // Re-run the effect when manifest changes
 
   const initCanvas = (canvasId, fromStart) => {
-    const {
-      isMultiSource,
-      sources,
-      tracks,
-      canvasTargets,
-      mediaType,
-      canvas,
-      error,
-    } = getMediaInfo({
-      manifest,
-      canvasIndex: canvasId,
-      srcIndex,
-    });
-    setIsVideo(mediaType === 'video');
-    manifestDispatch({ canvasTargets, type: 'canvasTargets' });
-    manifestDispatch({
-      canvasDuration: canvas.duration,
-      type: 'canvasDuration',
-    });
-    manifestDispatch({
-      isMultiSource,
-      type: 'hasMultipleItems',
-    });
-    // Set the current time in player from the canvas details
-    if (fromStart) {
-      if (canvasTargets?.length > 0) {
-        playerDispatch({ currentTime: canvasTargets[0].altStart, type: 'setCurrentTime' });
-      } else {
-        playerDispatch({ currentTime: 0, type: 'setCurrentTime' });
+    try {
+      const {
+        isMultiSource,
+        sources,
+        tracks,
+        canvasTargets,
+        mediaType,
+        canvas,
+        error,
+      } = getMediaInfo({
+        manifest,
+        canvasIndex: canvasId,
+        srcIndex,
+      });
+      setIsVideo(mediaType === 'video');
+      manifestDispatch({ canvasTargets, type: 'canvasTargets' });
+      manifestDispatch({
+        canvasDuration: canvas.duration,
+        type: 'canvasDuration',
+      });
+      manifestDispatch({
+        isMultiSource,
+        type: 'hasMultipleItems',
+      });
+      // Set the current time in player from the canvas details
+      if (fromStart) {
+        if (canvasTargets?.length > 0) {
+          playerDispatch({ currentTime: canvasTargets[0].altStart, type: 'setCurrentTime' });
+        } else {
+          playerDispatch({ currentTime: 0, type: 'setCurrentTime' });
+        }
       }
-    }
 
-    setPlayerConfig({
-      ...playerConfig,
-      error,
-      sources,
-      tracks,
-    });
-    updatePlayerSrcDetails(canvas.duration, sources, canvasId, isMultiSource);
-    setIsMultiSource(isMultiSource);
+      setPlayerConfig({
+        ...playerConfig,
+        error,
+        sources,
+        tracks,
+      });
+      updatePlayerSrcDetails(canvas.duration, sources, canvasId, isMultiSource);
+      setIsMultiSource(isMultiSource);
 
-    setCIndex(canvasId);
-    error ? setReady(false) : setReady(true);
+      setCIndex(canvasId);
+      error ? setReady(false) : setReady(true);
+    } catch (error) {
+      showBoundary(error);
+    }
```

Comment:
Does it work to rename this variable here?  I think it is a bit confusing since an `error` variable already exists inside the try/catch block.
```suggestion
    } catch (e) {
      showBoundary(e);
    }
```